src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -1810,6 +1873,10 @@ JoinClauseList(List *whereClauseList)
  * subqueries but differently from the outermost query, they are run on a copy
  * of parse tree and changes do not get persisted as modifications to the original
  * query tree.
+ *
+ * Also this function adds SubLinks to the baseQualifierList when they appear on
```

Comment:
We didn't change anything. But, wanted to explicitly mention that if sublinks exist in the `WHERE` clause, they'll be added to the qualifiers list.